conbench/app/_plots.py

```diff
@@ -450,13 +452,19 @@ def gen_js_callback_tap_detect_unselect(source: bokeh.models.ColumnDataSource):
     )
 
 
-def gen_js_callback_click_on_glyph_show_run_details(repo_string):
+def gen_js_callback_click_on_glyph_show_run_details(repo_url):
     """
     Dynamically manipulate Bootstrap panel conbench-histplot-run-details`.
 
     Show run details if a corresponding glyph was clicked in the plot.
 
     Warning: requires manual testing, not covered by tests/CI.
+
+    Note(JP): `repo_url` really is expected to be a URL and we have done quite
+    a bit of work in the last months to drive towards consistency, but at the
+    time of writing this sentence I am not sure if it really always is a URL.
```

Comment:
Yes, this is guaranteed to always be a URL.

https://github.com/conbench/conbench/blob/cf7931fbb415809bc3651f7f74692e09068468ac/conbench/entities/benchmark_result.py#L86

Reply:
Ha, I could have known! :) Thanks, adjusted comment.